content/en/serverless/step_functions/installation.md

```diff
@@ -224,11 +157,9 @@ If you have not yet instrumented your Lambda functions to send traces, you can [
 {{% /tab %}}
 {{< /tabs >}}
 
-## Enable enhanced metrics
-
-Datadog generates [enhanced metrics][8] from collected Cloudwatch logs. To enable this, add a `DD_ENHANCED_METRICS` tag to each of your Step Functions and set the value to `true`.
+## Link Step Functions with your AWS Lambda traces
 
-Enhanced metrics are automatically enabled if you enable traces.
+See [link Step Functions traces with Lambda traces][11]. Ensure that you have also [set up Serverless Monitoring for AWS Lambda][10].
 
 ## Enable tracing
```

Comment:
🤚  Sorry to add a late comment. But I do have a major concern here. 
It's a bit shame to admit that I actually made a mistake around here. 😊 
I went to see how to enable tracing first and assumed that all the necessary steps are included below. As a result, I missed the `Link step function with lambda` above.  
My other comments are all about the same concern but they are less critical. Here I want to suggest a change to add a NOTE to let the customer know that there's a dangling part of enabling tracing fully.

```suggestion
## Enable tracing
Note: There are extra steps needed if you want to see the step function spans and lambda spans connected in the same trace. See the section above.
```